build_adsbrain.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# apt-get update
+# apt-get install libssl-dev
+# apt-get install rapidjson-dev
+# nv-hostengine -t
+# distribution=$(. /etc/os-release;echo $ID$VERSION_ID | sed -e 's/\.//g')
+# wget https://developer.download.nvidia.com/compute/cuda/repos/$distribution/x86_64/cuda-keyring_1.0-1_all.deb
+# dpkg -i cuda-keyring_1.0-1_all.deb
+# apt-get install -y datacenter-gpu-manager
+# apt install libre2-dev
+# apt-get install -y cl-base64
+# apt-get install -y libb64-dev
+
+# rm -rf checksum/
+# rm -rf cmake_build
+# rm -rf opt/
+# rm -rf python
+# rm -rf tritonserver/
```

Comment:
if they are related to build? if not, remove

Reply:
yes, they are needed to install dependencies.